packages/algob/src/lib/algo-operator.ts

```diff
@@ -226,6 +236,76 @@ export class AlgoOperatorImpl implements AlgoOperator {
     };
   }
 
+  /**
+   * Description: Function to deploy Stateful Smart Contract
+   * @param approvalProgram name of file in which approval program is stored
+   * @param clearProgram name of file in which clear program is stored
+   * @param flags         SSCDeploymentFlags
+   * @param payFlags      TxParams
+   * @param txWriter
+   */
```

Comment:
Please add a link to the stateful sdk's here. [https://developer.algorand.org/docs/features/asc1/stateful/sdks/#application-methods]()

Reply:
then we will have to add links for all the sdk functions, this may be confusing because we have SPEC for clarity on what parameters to pass. what do you think ?

Reply:
Ok, let's add it in SPEC